core/vm/interpreter.go

```diff
@@ -133,13 +137,23 @@ func (in *Interpreter) Run(snapshot int, contract *Contract, input []byte) (ret
 	for atomic.LoadInt32(&in.evm.abort) == 0 {
 		// Get the memory location of pc
 		op = contract.GetOp(pc)
+		logged = false
 
 		// get the operation from the jump table matching the opcode
 		operation := in.cfg.JumpTable[op]
 		if err := in.enforceRestrictions(op, operation, stack); err != nil {
 			return nil, err
 		}
 
+		if in.cfg.Debug {
+			pcCopy = uint64(pc)
+			gasCopy = uint64(contract.Gas)
+			stackCopy = newstack()
+			for _, val := range stack.data {
+				stackCopy.push(val)
```

Comment:
Ah, so apparently the 63/64-th handling does not operate on the actual `bigint` already on the stack, but places a new one there, so your fix works (even though it's a bit brittle).